tools/pot/docs/SimplifiedMode.md

```diff
@@ -8,7 +8,26 @@ Simplified mode is designed to make data preparation for the model optimization
 
 ## Usage
 
-To use the Simplified mode, prepare the data and place it in a separate folder. No other files should be present in this folder. There are two options to run POT in the Simplified mode:
+To use the Simplified mode, prepare the data and place it in a separate folder. No other files should be present in this folder.
+
+To apply optimization when there is only a model and no data is available. It is possible to generate a synthetic dataset using Dataset Management Framework (Datumaro) available on [GitHub](https://github.com/openvinotoolkit/datumaro). Currently, data generation is available only for Computer Vision models, it can take time in some cases.
+
+Install Datumaro:
+
+``` bash
+pip install datumaro>=0.4.0.1
```

Comment:
Do we really need to specify >=? The latest version will be installed by default and it already includes the functionality.

Reply:
No, I canremove it.
I added version to emphasize that the feature is available from release 0.4.0.1.

Reply:
We should if the synthetic images generation feature is not from the beginning of datumaro.

Reply:
I got confused. No, we should not specify the version in the readme file. It should be a part of the requirements.txt file.

Reply:
To tell the truth, it would be much more valuable if it is a sample that demonstrates e2e flow of data-free quantization with:
- readme
- requirements.txt
- main.py

But I suggest having it in the NNCF rather then here.